src/layers/SchematicLayer.ts

```diff
@@ -375,6 +432,28 @@ export class SchematicLayer<T extends SchematicData> extends PixiLayer<T> {
           this.drawCementPlug(obj, casings, holeSizes);
         }
       });
+
+    this.internalLayerVisibility.pAndALayerId &&
+      remainingPAndA.forEach((obj) => {
+        if (isPAndASymbol(obj)) {
+          const symbolRenderObject = this.prepareSymbolRenderObject(obj);
+          this.drawSymbolComponent(symbolRenderObject);
+        }
+        if (isCementPlug(obj)) {
+          this.drawCementPlug(obj, casings, holeSizes);
+        }
+      });
+
+    shouldStartAtCasingDiameter.forEach((perforation) => {
+      const perfShape = this.createPerforationShape(perforation, casings, holeSizes).map((p) => {
+        if (perforation.subKind === 'Open hole frac pack') {
+          return { ...p, diameter: p.diameter * 4 };
+        }
+        return { ...p, diameter: p.diameter };
+      });
+      const otherPerforations = perforations.filter((p) => p.id !== perforation.id);
+      this.drawComplexRope(perfShape, this.createPerforationTexture(perforation, perfShape, otherPerforations));
+    });
```

Comment:
I reckon this should be rendered on top of everything else?
Can this be moved to it's own method? 😄